theseus_gui/src/pages/Browse.vue

```diff
@@ -818,6 +868,9 @@ const showLoaders = computed(
 
 .search-container {
   display: flex;
+  height: 100%; /* takes up only the necessary height */
+  overflow-y: auto;
+  scroll-behavior: smooth;
```

Comment:
why is this needed? iirc it already is scrollable

Reply:
see disc